package.json

```diff
@@ -1,10 +1,26 @@
 {
-  "name": "usine-distribuee",
-  "version": "0.1.0",
-  "repository": "git@github.com:CaenCamp/usine-distribuee.git",
-  "license": "GPL-3.0",
-  "private": true,
-  "workspaces": [
-    "apps/*"
-  ]
+	"name": "usine-distribuee",
+	"version": "0.1.0",
+	"repository": "git@github.com:CaenCamp/usine-distribuee.git",
+	"license": "GPL-3.0",
+	"private": true,
+	"workspaces": [
+		"apps/*"
+	],
+	"scripts": {
+		"lint": "prettier --check \"./**/*.js\"",
+		"lint:fix": "prettier --write \"./**/*.js\""
```

Comment:
I found more clear to name it `format` instead of `lint:fix`.

```
"lint": "eslint .",
"format": "prettier --write --check \"**/*.+(js|md|json)\""
```

It's good practice to have different format action between CI and local env.
```
"format": "prettier --write \"**/*.+(js|md|json)\"",
"format:ci": "prettier --check \"**/*.+(js|md|json)\"",
```